pyhyp/pyHyp.py

```diff
@@ -384,25 +384,24 @@ def __init__(self, comm=None, options=None, debug=False):
         # Set the possible MPI Intracomm
         if comm is None:
             comm = MPI.COMM_WORLD
-        self.comm = comm
 
         # Default options for hyperbolic generation
         defOpts = self._getDefaultOptions()
 
+        # Use supplied options
+        if options is None:
+            raise Error("The options = keyword argument is *NOT* optional. " "It must always be provided")
+
+        # Initialize the inherited BaseSolver
+        super().__init__(name, category, defOpts, options, comm=comm)
```

Comment:
What's the standard here in terms of using keyword arguments? Should we use `=`?

Reply:
I find it more clear to use the keywords if the arguments are keywords. I have updated the call accordingly.